### PR TITLE
docs: added README to all packages

### PR DESCRIPTION
**Pull request by tjwelde**
## fixes KILTProtocol/ticket#1679
This PR adds a simple README to all the packages.
 It also changes the title of the main README: https://github.com/KILTprotocol/sdk-js/blob/6af96005082f995859b1e3eaf8a0f5ec6f554864/packages/sdk-js/README.md

## How to test:
- Look at the rendered README file on github.

## Checklist:

- [x] I have verified that the code works
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: packages/did/README.md [hunk 1]
```diff
@@ -2,10 +2,24 @@
 
 ![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
 
-# Kilt DIDs
+# KILT DIDs
 
 A KILT Decentralised Identifier (DID) is a string uniquely identifying each KILT user that allows them to create CTypes, issue/collect attestations, and create delegation hierarchies.
 A DID can be thought of as a container of different keys that are all under the control of the same DID subject.
 
 For more information about KILT DIDs, please visit our [official documentation](https://kiltprotocol.github.io/docs/docs/sdk/core-feature/did).
```
Comment (Diiaablo95) [inline packages/did/README.md:10]: Should we also update the link to the doc to point to dev.kilt.io?

File: packages/sdk-js/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 Data sovereignty and interoperability
 
-# Mash-Net SDK
+# KILT JavaScript / TypeScript SDK
```
Comment (Diiaablo95) [inline packages/sdk-js/README.md:8]: ❤️

File: packages/types/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+[![](https://user-images.githubusercontent.com/39338561/122415864-8d6a7c00-cf88-11eb-846f-a98a936f88da.png)
+](https://kilt.io)
+
+![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
+
+# KILT Types
+
+This package provides the generally applicable TypeScript types for different mechanisms in the KILT SDK.
```
Comment (Diiaablo95) [inline packages/types/README.md:8]:
```suggestion
This package provides the generally applicable TypeScript types for different operations provided by the KILT SDK.
```

File: packages/types/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+This is also contains all the types, which someone would need to support for working with KILT credentials and DIDs not using the SDK.
```
Comment (Diiaablo95) [inline packages/types/README.md:10]:
```suggestion
The package also contains all the types, which someone would need to support for working with KILT credentials and DIDs not using the SDK.
```

File: packages/utils/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[![](https://user-images.githubusercontent.com/39338561/122415864-8d6a7c00-cf88-11eb-846f-a98a936f88da.png)
+](https://kilt.io)
+
+![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
+
+# KILT Utils
+
+This package provides different generic utility (and stateless) functions.
```
Comment (Diiaablo95) [inline packages/utils/README.md:8]: Do we really need to specify that they are stateless? What if that changes in the future?

**Review by Diiaablo95 — APPROVED** [review]
A few nitpicks, but otherwise looks good to me. Long time due!

File: packages/utils/README.md (hunk 5 above)
Comment (tjwelde) [reply packages/utils/README.md:8]: The utils package was specifically defined to only contain stateless functions, that's why I wanted to mention it here.

File: packages/types/README.md (hunk 3 above)
Comment (tjwelde) [reply packages/types/README.md:8]: Hmm, not sure if operations is enough here. I wanted to point out, that it contains all the base types you would also need to rebuild an SDK basically (while currently not knowing, if that is true)

Comment (tjwelde) [reply packages/types/README.md:8]: ah, but I say this in the next paragraph... so maybe it is fine to take your suggestion

File: packages/utils/README.md (hunk 5 above)
Comment (Diiaablo95) [reply packages/utils/README.md:8]: Ok did not know the backstory, so that's fine.

File: packages/types/README.md (hunk 3 above)
Comment (tjwelde) [reply packages/types/README.md:8]: a05286eff11b655e0641df1ce8938f24b4bad710

File: packages/types/README.md (hunk 4 above)
Comment (tjwelde) [reply packages/types/README.md]: a05286eff11b655e0641df1ce8938f24b4bad710

File: packages/types/README.md (hunk 3 above)
Comment (tjwelde) [reply packages/types/README.md]: 5fed91be8f69ca7a08bf7cad96755c93aec05f74

**Review by Diiaablo95 — APPROVED** [review]
👍

**Pull request merged**